coffee-grpc/coffee-grpc-client-extension/src/main/java/hu/icellmobilsoft/coffee/grpc/client/config/GrpcClientConfig.java

```diff
@@ -66,6 +66,16 @@ public class GrpcClientConfig implements IGrpcClientConfig {
      */
     public static final String MAX_INBOUND_METADATA_SIZE = "maxInboundMetadataSize";
 
+    /**
+     * gRPC client request log maximum size in characters, default is 1000
+     */
+    public static final String REQUEST_LOG_SIZE = "requestLogSize";
```

Comment:
class javadocja tartalmazza a minta konfigot, oda is leccives ird bele

Reply:
Javítva